components/automate-gateway/api/compliance/reporting/reporting.proto

```diff
@@ -92,8 +109,15 @@ service ReportingService {
 			resource: "compliance:reporting:profiles"
 			action: "compliance:reportProfiles:list"
 		};
+		option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+			description: "List all profiles optionally using filters";
```

Comment:
i wonder if there are words we can use here to make it clear that these are profiles as in profile results from a scan..?